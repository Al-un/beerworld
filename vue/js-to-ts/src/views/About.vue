<template>
  <div class="about">
    <h1>This is an about page with : {{ text }}</h1>
  </div>
</template>

<script>
export default {
  // Error: options order
  data() {
    return { text: "Some text" };
  },
  name: "about", // Error: should be PascalCased
};
</script>
