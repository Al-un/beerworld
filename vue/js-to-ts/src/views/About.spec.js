import { mount } from "@vue/test-utils";

import About from "./About";

let wrapper;

describe("views/About", () => {
  it("renders and match snapshot", () => {
    wrapper = mount(About);

    expect(wrapper).toMatchSnapshot();
  });
});
