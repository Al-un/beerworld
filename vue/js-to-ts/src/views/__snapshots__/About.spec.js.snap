// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`views/About renders and match snapshot 1`] = `
<div class="about">
  <h1>This is an about page with : Some text</h1>
</div>
`;
